lib/instana/config.rb

```diff
@@ -39,6 +39,7 @@ def initialize
       @config[:excon]              = { :enabled => true }
       @config[:nethttp]            = { :enabled => true }
       @config[:'rest-client']      = { :enabled => true }
+      @config[:'grpc']             = { :enabled => true }
```

Comment:
I think we can change this globally to just `:grpc`.  The quotes were only needed when there is a dash or underscore in the name.